### PR TITLE
[wampcc] Add new port

### PR DESCRIPTION
**Pull request by sergluka**
Related to #7928

### CODE REVIEW

**Review by cbezault — CHANGES_REQUESTED** [review]

File: ports/wampcc/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    set(USE_SHARED_RUNTIME ON)
+else()
+    set(USE_SHARED_RUNTIME OFF)
+endif()
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DBUILD_STATIC_LIBS:BOOL=${USE_STATIC_RUNTIME}
+        -DBUILD_SHARED_LIBS:BOOL=${USE_SHARED_RUNTIME}
```
Comment (cenit) [inline ports/wampcc/portfile.cmake]: Why this mixture between libs and runtime? Is it ok? 

BUILD_SHARED_LIBS is a standard CMake symbol and properly defined already by the vcpkg toolchain. If the "mixture" of libs on the left and crt on the right of the = symbol is by design, which means for example that the library when built as static works only with static CRT, that's a different problem and requires a different approach, similar to the vcpkg_check_linkage(ONLY_STATIC_LIBRARY) at the beginning

Comment (SergeyLukashevich) [reply ports/wampcc/portfile.cmake]: Extra symbols are removed

File: ports/wampcc/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        -DBUILD_UTILS:BOOL=OFF
```
[the remark this answers is not in the document]
Comment (SergeyLukashevich) [reply ports/wampcc/portfile.cmake]: Utils and examples feature has been added. Tests build is broken in a recent version.

File: ports/wampcc/CONTROL [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Source: wampcc
+Build-Depends: openssl, libuv, jansson
+Version: 1.6
```
Comment (dan-shaw) [inline ports/wampcc/CONTROL]: This appears to be a rolling-release port, so we want the version formatted as `YYYY-MM-DD` (the date that the commit was accessed by you).

**Review by cbezault — APPROVED** [review]

**Pull request merged**